Packs/CreatePlbkDoc/.pack-ignore

```diff
@@ -2,6 +2,7 @@
 ignore=RM106
 
 [file:pack_metadata.json]
-ignore=PA129
-
+ignore=PA129, PA126
 
+[file:CreatePlbkDoc.yml]
+ignore=DO106
```

Comment:
Need to update the docker image in the script yml file.
This error cannot be ignored.
```suggestion
```